packages/integration-tests/src/test-project-setup/conversation_handler_project.ts

```diff
@@ -168,37 +343,32 @@ class ConversationHandlerTestProject extends TestProjectBase {
         systemPrompt: 'You are helpful bot.',
       },
     };
+    const response = await this.executeConversationTurn(
+      event,
+      customConversationHandlerFunction,
+      apolloClient
+    );
+    // Assert that tool was used.
+    assert.match(response.content, /75/);
```

Comment:
why 75 here?

Reply:
It's separate scenario, so I wanted to use different value to make sure that they don't accidentally cross wire.

Reply:
Nvm i see.. we are just matching the response json?

Reply:
correct, and we use mocked appsync schema to capture the response so `response.content` contains serialized json. at least for now while other components (schema builder) are in works (but we may choose to retain mocked schema to be independent from these components).

the goal of e2e test at this point is to assert that roundtrip data exchange happened and details of emitted. responses are captured in unit tests mostly.